lib/apiservers/engine/backends/container.go

```diff
@@ -770,13 +783,60 @@ func UnmapPorts(hostconfig *containertypes.HostConfig) error {
 			return err
 		}
 
+		// bridge-to-bridge pin hole for traffic from containers for exposed port
+		if err = interBridgeTraffic(portmap.Unmap, p.strHostPort, "", "", ""); err != nil {
+			return err
+		}
+
 		// update mapped ports
 		delete(containerByPort, p.strHostPort)
 		log.Debugf("unmapped port %s", p.strHostPort)
 	}
 	return nil
 }
 
+// interBridgeTraffic enables traffic for exposed port from one bridge network to another
```

Comment:
Does this code really need to be in container.go?  Container.go and all the backend router files should really just be REST routing.